content/docs/ops/runbook/troubleshooting-kubernetes.md

```diff
@@ -92,14 +96,21 @@ rescheduled:
 kubectl delete pod x0e41a06287f54-master-1362982585-aamfj
 ```
 
+### Other useful Kubernetes commands
+
+#### Stop entire Kubernetes deployment
+```shell
+bosh-cli stop kubernetes
+```
+
 #### NodePorts
 Currently services are deployed via NodePort, which means that Kubernetes generates a temporary random port, and assigns that port to each Master node.  This port will redirect to the containers exposed service port, for example 9200
 
   - Node Port 31000
   - Service Port 9200
 
-  ## Both of these return the same host via iptables:
-  ```
-  curl http://0.master.default.kubernetes.bosh
-  curl http://1.master.default.kubernetes.bosh
-  ```
\ No newline at end of file
+Both of these return the same host via iptables:
+```
+curl http://0.master.default.kubernetes.bosh
```

Comment:
I see that this is just an indentation change, but:

* These DNS names are wrong
* The kubernetes machines list aws dns before bosh dns, so these names don't resolve
* If we query bosh dns, the names don't resolve to the same address.

WDYT about deleting this block?

#writeonlydocs